torch/distributed/elastic/utils/distributed.py

```diff
@@ -113,6 +113,23 @@ def _check_full_rank(store, world_size, timeout):
 
 
 def get_free_port():
+    """
+    Returns an unused port on localhost.
+
+    This function finds an unused port on localhost by opening to socket to bind
+    to a port and then closing it.
+
+    Returns:
+        int: an unused port on localhost
+
+    Example:
+        >>> get_free_port()
```

Comment:
Please add `# xdoctest: +SKIP("Nondeterministic")` to fix the doctest errors. See https://github.com/pytorch/pytorch/blob/8892ddaaccf7f07d64e2e819d868c0e95bc53e74/torch/autograd/forward_ad.py#L82 as an example

Reply:
Thank you for the suggestion, done now.